src/invisible/explanation/data-streams.md

```diff
@@ -0,0 +1,87 @@
+---
+description: TODO
+---
+
+# Data Streams
+Pupil Invisible glasses include a number of sensors providing different types of data. Some data streams are available in real-time (see [real-time API](/invisible/tutorials/real-time-api)), while others are computed in Pupil Cloud post hoc after uploading. All data is fully accessible and can be downloaded from Pupil Cloud in convenient formats.
+
+## Eye Videos
+Pupil Invisible glasses feature two eye cameras, one for each eye. They are fully embedded into the frame. The sensors record IR video at 200 Hz and 192x192px resolution. Right next to each camera is an IR LED, which guarantees good illumination of the eye in dark environments.
+
+<div style="display:flex;justify-content:center;" class="pb-4">
+  <v-img
+    :src="require('../../media/invisible/PI-Camera_annotation-white.jpg')"
+    max-width=80%
+  >
+  </v-img>
+</div>
+
+Below you can find a collection of example image pairs recorded with different subjects:
+
+<div style="display:flex;justify-content:center;" class="pb-4">
+  <v-img
+    :src="require('../../media/invisible/example-eye-images.jpg')"
+    max-width=100%
+  >
+  </v-img>
+</div>
+
+## Gaze
+While recording, the Pupil Invisible Companion device is calculating gaze data in real-time. The exact framerate of this signal depends on the model of phone you use as Companion device. On a OnePlus 6 device, the framerate is around 50 Hz, while on a OnePlus 8 it is around 65 Hz. Other apps running simultaneously on the phone may decrease the framerate.
+
+After a recording is uploaded to Pupil Cloud, the gaze data is automatically re-computed at the full framerate of 200 Hz.
+
+<div style="display:flex;justify-content:center;" class="pb-4">
+  <v-img
+    :src="require('../../media/invisible/pi-gaze-coordinate-diagram.jpg')"
+    max-width=80%
+  >
+  </v-img>
+</div>
+
+The gaze data is output in pixel space of the scene camera image, which has a resolution of 1088x1080px. The origin is in the top-left corner of the image.
+
+You can find a high-level description as well as a thorough evaluation of the accuracy and robustness of the gaze estimation algorithm in our [white paper](https://arxiv.org/pdf/2009.00508).
+
+## Fixations
+The two primary types of movement the eye is performing are fixations and saccades. During fixation, the eye is fixated on a specific point in the environment. A saccade is a very quick movement where the eye is jumping from one fixation to the next. Properties like the fixation duration are correlated with cognitive processes like e.g. cognitive load.
+
+Fixations are calculated automatically in Pupil Cloud after uploading a recording. They are available as part of all enrichment downloads. The downloads for gaze mapping enrichments ([Reference Image Mapper](/invisible/explanation/enrichments/#reference-image-mapper), [Marker Mapper](/invisible/explanation/enrichments/#marker-mapper)) also include "mapped fixations".
```

Comment:
Not strictly ALL enrichments, as Gaze Overlay (if it remains an enrichment) has no associated .csv, and overlays gaze, not fixations